main.go

```diff
@@ -58,6 +59,19 @@ func setupLogging() {
 	log.Debug().Msgf("Log level set to %q", logLevel)
 }
 
+func saveLogsToFile(confFileLogLocation string) {
+
+	logFileWithRotation := &lumberjack.Logger{
+		Filename:   confFileLogLocation,
+		MaxSize:    100, // Max size in megabytes
```

Comment:
I like log rotation, but are these parameters something that should be configurable by the user?

Reply:
Fixed. Added config parameters, that users can utilize to provide log rotation handles